setup.py

```diff
@@ -49,7 +49,9 @@
         "pyopenssl>=20.0.0",
         "PyInstaller>=4.0",
         "PyYAML>=5.3.1",
-        "requests>=2.24.0",
```

Comment:
I know its not related to PR and can be done i a different one but maybe it worth considering reading the requirments.txt file here and giving it to install_requires variables. So the requirments will appear in one place